.codecov.yml

```
@@ -0,0 +1,12 @@
+comment: off
+github_checks:
+  annotations: false
+coverage:
+  status:
+    patch:
+      default:
+        target: 95%
+        only_pulls: true
+    project:
+      default:
+        threshold: 1%
```

```suggestion
        threshold: 1%

```
add newspace at the end